src/util_types/mutator_set/archival_mutator_set.rs

```diff
@@ -88,8 +88,8 @@ where
         self.accumulator().await.hash()
     }
 
-    /// Apply a list of removal records while keeping a list of mutator set membership proofs
-    /// up-to-date
+    /// Apply a list of removal records while keeping a list of mutator set
+    /// membership proofs up-to-date.
     pub async fn batch_remove(
```

Comment:
I don't like how this function uses the mutator set accumulator as a library. Seems sloppy but the changes look correct to me. Though, it's, due to multiple rewrites, a rather confusing implementation.